pkg/swagger/typewalker.go

```diff
@@ -117,25 +117,27 @@ func (tw *typeWalker) schemaFromType(t types.Type, deps map[*types.Named]struct{
 		}
 
 	case *types.Struct:
+
 		for i := 0; i < t.NumFields(); i++ {
 			field := t.Field(i)
-
-			nodes, _ := tw.getNodes(field.Pos())
-			node := nodes[1].(*ast.Field)
-			tag, _ := strconv.Unquote(node.Tag.Value)
-
-			name := strings.SplitN(reflect.StructTag(tag).Get("json"), ",", 2)[0]
-			if name == "-" {
-				continue
+			if field.Exported() {
```

Comment:
time.Time causes this.

Reply:
Should not be needed now

Reply:
> Should not be needed now

What do you mean? We need to revert this change?